src/hashutil.h

```diff
@@ -6,35 +6,46 @@
 #include <sys/types.h>
 
 #include <string>
+#include <fstream>
 
 #include <random>
 
 namespace hashing {
-// See Martin Dietzfelbinger, "Universal hashing and k-wise independent random
-// variables via integer arithmetic without primes".
-/*
-class TwoIndependentMultiplyShift {
 
-  unsigned __int128 multiply_, add_;
+inline size_t sysrandom(void *dst, size_t dstlen) {
```

Comment:
Why not use ::std::random_device ?

Reply:
We can use it. 
I used your previuos version, and uncomment the implementation of the "TwoIndependentMultiplyShift" hashing, which (I think) I need.